doc/maintainers.md

```diff
@@ -1,5 +1,22 @@
 # Maintainers
-This page describes the current module, library, and extension maintainers of Ruby.
+This page describes the current branch, module, library, and extension maintainers of Ruby.
+
+## Branch Maintainers
+
+A branch maintainer is responsible for backporting commits into stable branches
+and publishing Ruby patch releases.
+
+### Ruby 3.3
+
+* NARUSE, Yui (nurse)
```

Comment:
I inferred this from https://bugs.ruby-lang.org/issues/20085#note-12, but couldn't otherwise find the information.